### ISSUE TITLE
Consuming design(-related) media e.g. books, magazines, podcasts at work or off-time

### ISSUE THREAD

**Issue by patrickmrx** [designdetails/designdetails#540]
All my career I consumed media related to design. I always found podcasts to be very helpful but also read a lot of design books (The Design of Everyday Things, Hooked, you name it), and I still do. Being a Designer for almost seven years now, I almost only listened and read during off-time. I never really thought about doing this while at work because... it didn't feel right? I don't know, but the idea of sitting next to my coworkers reading a book while they're “working regularly“ felt weird.
However, working from home for almost a year, I gave it a second thought. I love listening to design podcasts but I mainly do it to get better at work.. So why would I listen to them first thing in the morning while getting ready and not during work time? I mean, if it's podcasts without a real educational purpose, fine. But with most podcasts, I'm learning something new. Basically, I'm educating myself. Considering this, it almost feels crazy to do this during off-time.
So I cut out design podcasts and lectures from off-time and planned to consume them at a certain time while working. That freed up lots of time for non-work-related media, and I love it. However, I don't really make time for reading and educating myself, it all gets lost as I'm in work mode and the next time I look at the clock it's 5 pm.

TL;DR
I'm struggling to find the right balance between consuming design media and educating myself and regular work. How do you do it? Do you consume design-related content during your off-time? Do you free up time during your workdays?

**Comment by brianlovin** [comment]
Hey! We talked a lot about design content overload in last week's Sidebar. Hope it's helpful here! https://designdetails.fm/episodes/rKjF2y1A

**Comment by patrickmrx** [comment]
Yes, listened and enjoyed ✅
Thanks!
